csv_parser/json_schema2xsd/src/main/java/json_schema2xsd/App.java

```diff
@@ -31,8 +36,26 @@ public static void main(String[] args) {
         Logger logger = Logger.getLogger(App.class.getName());
         List<String> regexErrors = new ArrayList<>();
         
-        for (String schema : Arrays.asList("EMSI", "RC-DE", "RC-EDA", "RC-REF", "RS-EDA", "RS-INFO", "GEO-RES", "GEO-REQ", "GEO-POS", "RS-ERROR", "RS-RI",
-                "RS-DR", "RS-RR", "RS-RPIS", "RS-EDA-MAJ", "RS-SR", "TECHNICAL", "TECHNICAL_NOREQ", "RS-URL", "RS-ER", "RS-BPV")) {
+        //Get the schemas.yaml file from resource folder
+        InputStream schemaStream = App.class.getResourceAsStream("/schemas.yaml");
+        assert schemaStream != null;
+        InputStreamReader schemaReader = new InputStreamReader(schemaStream);
+        Map<String, List<HashMap<String,String>>> yamlMap = new HashMap<>();
+        try (Reader reader = new BufferedReader(schemaReader)) {
+            Yaml yaml = new Yaml();
+            yamlMap = yaml.load(reader);
+        } catch (IOException e) {
+            logger.severe("Error while reading schemas.yaml: " + e);
+            System.exit(1);
+        }
+        List<HashMap<String,String>> schemas = yamlMap.get("schemas");
+        
+        for (HashMap<String, String> scheMap : schemas) {
```

Comment:
```suggestion
        for (HashMap<String, String> schemaConfig : schemas) {
```
plus parlant ? même si joli la contraction scheMap ahah